### DISCUSSION TITLE
A way to hide some browsers?

### DISCUSSION THREAD

**Discussion by boldlion** [will-stone/browserosaurus#533] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues
- [X] I have searched the [discussions forum](https://github.com/will-stone/browserosaurus/discussions)
- [X] I understand this form is for reporting bugs, and not for requesting [support for new browsers or apps](https://github.com/will-stone/browserosaurus/blob/master/guide/supporting-a-browser-or-app.md).

### Current Behaviour

1. It detects apps like Spotify and Zoom as "browsers"
2. There should be a way to hide false positives (if I can call them that) but I cant find a way
![Screenshot 2022-05-02 at 3 49 53 PM](https://user-images.githubusercontent.com/43475912/166236401-1771d2be-cad5-4ac1-a2de-00318b2d7066.png)

### Expected Behaviour

ability/button to hide some browsers that you don't want to be included in the list 

### Steps To Reproduce

1. Open the app
2. Re-scan it and find all browsers
3. No way to remove those that you don't want included

### Browserosaurus version

18.1.0

### macOS version

12.3.1

### CPU Architecture

ARM

**Comment by will-stone** [comment]
Hi. Thanks for using Browserosaurus. I converted this to discussion as it’s not a bug. Please see this discussion:  https://github.com/will-stone/browserosaurus/discussions/461

**Comment by stevemckenzie** [comment]
Please bring this feature back. You use to have it 🙃

**Reply by will-stone** [reply]
Sorry, nah. Covered here: https://github.com/will-stone/browserosaurus/discussions/461

**Reply by stevemckenzie** [reply]
lol ok, that works for me, thanks!
